src/base/net/stratum/SelfSelectClient.cpp

```diff
@@ -201,6 +203,9 @@ void xmrig::SelfSelectClient::submitBlockTemplate(rapidjson::Value &result)
     Document doc(kObjectType);
     auto &allocator = doc.GetAllocator();
 
+    m_blocktemplate = Json::getString(result,kBlocktemplateBlob);
+    m_targetdiff = Json::getUint64(result, kDifficulty);
```

Comment:
As this is the actual blockchain diff target, I'd suggest renaming this variable to something like `m_blockdiff`.